client/packages/electron/src/electron.ts

```diff
@@ -446,6 +456,40 @@ const helpMenu: MenuItemConstructorOptions = {
         );
       },
     },
+    {
+      label: 'Clear Data',
+      click: () => {
```

Comment:
added the dev menu items as we've had some issues raised with desktop and it's impossible to troubleshoot. May remove again once we've got to the bottom of things.